platform/ui/src/components/userPreferencesModal/AboutModal.js

```diff
@@ -69,13 +97,15 @@ export class AboutModal extends Component {
             >
               Visit the forum
             </a>
+            {` `}
```

Comment:
@dannyrb Is this an ESLint thing? Why do these keep getting inserted?

Reply:
It assumes you want to preserve a space. Newline? No space. This thing? A space.